soql-standalone-parser/src/test/scala/com/socrata/soql/parsing/ParserTest.scala

```diff
@@ -218,7 +218,7 @@ class ParserTest extends WordSpec with MustMatchers {
       println("test 1")
       val x = parseFull("select count(distinct column)")
       x.selection.expressions.head.expression.asInstanceOf[FunctionCall].functionName.name must be ("count_distinct")
-      x.selection.expressions.head.expression.toString must be ("COUNT(DISTINCT `column`)")
+      x.selection.expressions.head.expression.toString must be ("count(DISTINCT `column`)")
```

Comment:
huh, this is not a thing I would have expected to change?

Reply:
It is intentional changed.  You didn't see where? In functioncall.doc

Reply:
oh! yeah I do see it. but why?

Reply:
More consistent with function name style.  I was gonna to give reason but forgot.